raster/src/test/scala/geotrellis/raster/io/geotiff/writer/GeoTiffWriterSpec.scala

```diff
@@ -54,6 +54,52 @@ class GeoTiffWriterSpec extends AnyFunSpec with Matchers with BeforeAndAfterAll
       actual should be (expected)
     }
 
+    it("should write GeoTiff with correct layout (tiled vs striped)") {
```

Comment:
I feel like we could split tests into two: 

* Tiled => Striped conversion (the source is read from a tiled TIFF file; let's create this one via GDAL, so it's a given input)
* Striped => Tiled conversions (the source is read from a striped TIFF file; let's create this one via GDAL, so it's a given input)

For two categories of Tiffs:

* Singleband
* Multiband 

Total +4 separate tests (a bit easier to read, and less complex in general).